qubes/storage/reflink.py

```diff
@@ -372,25 +400,38 @@ def _replace_file(dst):
 _remove_file = functools.partial(
     qubes.utils.remove_file, log_level=logging.INFO)
 
-def _make_dir(path):
-    ''' mkdir path, ignoring FileExistsError; return whether we
-        created it.
-    '''
```

Comment:
Ditto

Reply:
That one seemed pointless after polishing the function